nginx.conf

```diff
@@ -6,9 +6,9 @@ server {
         proxy_pass http://node-urdr:4242/;
     }
 
-    location /report {
-        proxy_pass http://node-urdr:4242/;
-    }
+#    location /report {
+ #       proxy_pass http://node-urdr:4242/;
+  #  }
```

Comment:
Is there a reason this is out-commented instead of removed?

Reply:
no its an error on my side we should delete this